packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -315,7 +335,8 @@ export const Stackbox = () => {
 
   const estimatedNumberOfOrders =
     Math.floor(
-      (endDateTime.getTime() - startDateTime.getTime()) /
+      (getDatePlusMinsOffset(endDateTime, ONE_MINUTE).getTime() -
+        startDateTime.getTime()) /
```

Comment:
Time should be added to startDateTime. Let's add 10 minutes to it.

Reply:
Something like this you mean?
```ts
    Math.floor(
      (endDateTime.getTime() -
        getDatePlusMinsOffset(startDateTime, TEN_MINUTES).getTime()) /
        frequencySeconds[frequency as FREQUENCY_OPTIONS]
    ) + INITAL_ORDER;
```
    
If I do this, we'll end up having less time slots for the given frequency
So for example: 
* Start: July 21th, at 12:00
* End: July 24th, at 12:00
* Frequency: daily
* 4 WXDAI to WETH

Should display "1 WXDAI daily for 3 days" (due to the initial order)
But instead, it will display this:
<img width="484" alt="Screenshot 2024-07-16 at 12 08 54" src="https://github.com/user-attachments/assets/a167df1b-e3a0-4d84-a1d3-444cf96bff77">

Reply:
If you create the stack, what are the stacks buying timestamps?

Reply:
We call the slots